### DISCUSSION TITLE
Financial Object & Reg Innovation SIGs: Discussion Topics Long List

### DISCUSSION THREAD

**Discussion by jgavronsky** [finos/open-regtech-sig#63] [General]
Hello, we would like to get input on topics of interest for the community, and invite relevant presenters to the SIG meetings.
Please suggest topics and presenters that are of interest to you. Topics can cover both Regulatory Innovation and Financial Objects (indicate which one is prevalent, as needed). Since there is frequent overlap between the two, we are commingling the topics list. (updated 1/4/24)
1. Post-Trade regulation compliance: stories from the field (RI)
2. T+1 reporting Requirements: a case for collaborative component development (RI)
3. ESG Data standardization: a project proposal
4. ISDA DRR Presentation (Feb, RI)
5. CDM for the Buy Side (Pictet presentation) (Feb, RI)
6. Standard Settlement Instructions - template build and potential digital authentication. FMSB Workstream. (FO)
7. SupTech: a market survey (Jane: can invite Cambridge SupTech Lab to present their [findings](https://lab.ccaf.io/state-of-suptech-report/))
8. [White House AI Executive Order](https://www.whitehouse.gov/briefing-room/statements-releases/2023/10/30/fact-sheet-president-biden-issues-executive-order-on-safe-secure-and-trustworthy-artificial-intelligence/): implications for FS (Jane: joint presentation/presenters with Zenith SIG?)
9. [FDTA - what's next?](https://www.sec.gov/files/2023-fdta-report.pdf) (Jane: update from the Data Foundation?)

**Comment by ffionwiggins** [comment]
Standard Settlement Instructions - template build and potential digital authentication. FMSB Workstream.

**Comment by eteridvalishvili** [comment]
Hello, I have compiled below some of the discussion items from the emails that were sent before the holidays. Please let me know if I have missed anything.

1. Cover the work being done with Financial Objects / Reg tech Council.  
2. Discuss the survey within the first  2-3 working days of the new year. Give a week to get the responses back and collate the results.  
3. Possible ISDA presentation on DRR in Feb. 
4. Need priorities from Morgan Stanley and Citi to socialize in the next meeting.
